packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -800,6 +806,696 @@ let rows = [
+  let {popoverRef = ref, triggerRef} = props;
+  let {popoverProps, underlayProps} = usePopover({
+    ...props,
+    popoverRef,
+    triggerRef
+  }, state);
+
+  return (
+    <Overlay>
+      <div {...underlayProps} style={{position: 'fixed', inset: 0}} />
+      <div
+        {...popoverProps}
+        ref={popoverRef}
+        style={{
+          ...popoverProps.style,
+          background: 'lightgray',
+          border: '1px solid gray'
+        }}>
+        <DismissButton onDismiss={state.close} />
+        {children}
+        <DismissButton onDismiss={state.close} />
+      </div>
+    </Overlay>
+  );
+}
+```
+
+</details>
+
+### Button
+
+The `Button` component is used in the above example to toggle the menu. It is built using the [useButton](useButton.html) hook, and can be shared with many other components.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+import {useButton} from '@react-aria/button';
+
+function Button(props) {
+  let ref = props.buttonRef;
+  let { focusProps, isFocusVisible } = useFocusRing();
+  let outline = isFocusVisible
+    ? '2px solid orange'
+    : 'none';
+  let {buttonProps} = useButton(props, ref);
+  return <button {...mergeProps(buttonProps, focusProps)} ref={ref} style={{...props.style, outline: outline}}>{props.children}</button>;
+}
+```
+
+</details>
+
+### Resizable table body
+
+Similar to `TableRowGroup` and `TableHeaderRow`, `TableRow` and `TableCell` only require minor style changes to
+accommodate the new `display` changes made in the `ResizableColumnsTable`. The changes to `TableRow` are made to extend the
+background of the row to the full width of its child cells. `TableCell` now receives its width from <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+and handles text overflow.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function ResizableTableRow({item, children, state}) {
+  // Same as previous TableRow implementation
+  ///- begin collapse -///
+  let ref = useRef();
+  let isSelected = state.selectionManager.isSelected(item.key);
+  let {rowProps, isPressed} = useTableRow({
+    node: item
+  }, state, ref);
+  let {isFocusVisible, focusProps} = useFocusRing();
+  ///- end collapse -///
+  return (
+    <tr
+      style={{
+        background: isSelected
+          ? 'blueviolet'
+          : isPressed
+            ? 'var(--spectrum-global-color-gray-400)'
+            : item.index % 2
+              ? 'var(--spectrum-alias-highlight-hover)'
+              : 'none',
+        color: isSelected ? 'white' : null,
+        outline: 'none',
+        boxShadow: isFocusVisible ? 'inset 0 0 0 2px orange' : 'none',
+        /*- begin highlight -*/
+        display: 'flex',
+        width: 'fit-content'
+        /*- end highlight -*/
```

Comment:
hmm a lot of the changes we are highlighting are purely style changes. Wonder if there is a way of simplifying this. Also, why does the table have a flex layout rather than table?

Reply:
display table has a lot of issues with respecting designated column sizes and cannot be horizontally scrolled.
For example, if you specify all columns with a min of 50px and then resize one column so it's huge and would cause horizontal scrolling, all the other columns will shrink to be smaller than the min specified.

Reply:
probably, but then you don't get the sticky headers if I recall correctly